mwdb/model/user.py

```diff
@@ -170,18 +170,18 @@ def _generate_token(self, user_fields, scope, expiration, **extra_fields):
     def _verify_token(token, fields, scope) -> Optional[Tuple["User", Optional[str]]]:
         data = verify_token(token, scope)
         if data is None:
-            return None
+            return None, None
```

Comment:
The returned value is not of the declared return type `tuple[User, str | None] | None`. I firmly believe that there should be no `None, None` case because the initial return type is simply easier to interpret – if a bare `None` value, then the user is _not found_, otherwise always a 2-element tuple with meaningful data.